### ISSUE TITLE
[INQUIRY] Non Compliant Trackers

### ISSUE THREAD

**Issue by zoracon** [ekr/draft-fossaceca-dult-finding#2]
(This is Alexis H. from EFF) 

I figure this was the best way to leave a comment or question

>RE: However, if an attacker were to make their own accessory that was generated the right beacon messages or modify an existing one, they could cause it to rotate the MAC address more frequently, thus evading detection algorithms. The attestation mechanism described in Section [TODO] is intended to mitigate this attack.

Can we specifically state with this draft that non compliant trackers be locked out from being able to utilize the crowd sourced FindMy networks? We can't stop more sophisticated trackers, but we can at least stop them from being able to use this network. 

Forgive me if this is addressed somewhere else.

**Comment by ekr** [comment]
Yes, that's the objective of the techniques to be described in [TODO]. It's TBD exactly what those techniques are and how well they will work.
